package/boot/grub2/grub2-efi/Makefile

```diff
@@ -0,0 +1,22 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=grub-efi
+
+include ../common.mk
+
+TAR_OPTIONS:= --transform 's/grub-${PKG_VERSION}/${PKG_NAME}-${PKG_VERSION}/' $(TAR_OPTIONS)
+
+PKG_BUILD_DEPENDS:=grub2-efi/host
+
+CONFIGURE_ARGS += --with-platform=efi
+HOST_CONFIGURE_ARGS += --with-platform=efi --program-suffix=-efi
+
+define Package/grub2-efi
+$(call Package/grub2/Default)
+DEPENDS:=@EFI_IMAGES +kmod-efi-vars
```

Comment:
There's no need for the @EFI_IMAGES dependency. Simply add HIDDEN:=1 to avoid making it selectable directly.